frontend/integration-tests/views/crud.view.ts

```diff
@@ -110,16 +115,17 @@ export const deleteRow = (kind: string) => (name: string) =>
         await $('input[placeholder="Enter name"]').sendKeys(name);
         break;
       default:
-        await browser.wait(until.presenceOf($('#confirm-action')));
+        await browser.wait(until.elementToBeClickable($('#confirm-action')));
         break;
     }
 
     await $('#confirm-action').click();
+    await browser.sleep(2000);
```

Comment:
Why do we have an arbitrary sleep?

Reply:
It must have been one of those "bad flake days" that I hit it and added that line. Dropping...